201-machine-learning-dependencies-role-assignment/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "type": "QuickStart",
+    "itemDisplayName": "User assigned identity role assignment template",
+    "description": "A template that creates role assignments of user assigned identity on resources that Azure Machine Learning workspace depends on",
+    "summary": "Demonstrates What role assignment are needed on the dependent resources of Azure Machine Learning user assigned identity enabled workspace to make the workspace functions as expected.",
+    "githubUsername": "zhaomuzhi",
+    "docOwner": "Blackmist",
+    "dateUpdated": "2020-12-03",
+    "environments": [
+        "AzureCloud"
```

Comment:
why not USGov?

Reply:
The UAI feature has not been enabled for sovereign cloud yet. We will enable it on service side later.